widget/entry.go

```diff
@@ -1560,6 +1560,9 @@ type entryContentRenderer struct {
 }
 
 func (r *entryContentRenderer) Destroy() {
+	if !fyne.CurrentApp().Settings().ShowAnimations() {
+		return
```

Comment:
Can this not leak if the user disables animations after one has started?

Reply:
Thanks for this. I had thought that, but loop forever is on so they won't end. I have removed the guard here so it will always try to stop.